sdk/python/feast/feature_store.py

```diff
@@ -998,6 +999,48 @@ def serve(self, port: int) -> None:
 
         feature_server.start_server(self, port)
 
+    @log_exceptions_and_usage
+    def upload_docker_image(self) -> None:
+        """Upload the docker image for the feature consumption server to the cloud."""
+
+        # TODO: add error checking and avoid hardcoding the region
+        repository_name = "feast-python-server-test"
+        feature_server_type = (
+            self.config.feature_server.type if self.config.feature_server else None
+        )
+        if feature_server_type == "aws_lambda":
+            import base64
+
+            import boto3
+            import docker
+            from botocore.exceptions import ClientError
+
+            docker_client = docker.from_env()
+            image_name = FEATURE_SERVER_IMAGE_FOR_TYPE[feature_server_type]
+            docker_client.images.pull(image_name)
+
+            ecr_client = boto3.client("ecr", region_name="us-west-2")
+            try:
+                ecr_client.create_repository(repositoryName=repository_name)
+            except ClientError:
+                pass
```

Comment:
Why pass? We need to handle the error. You can check if the error is because it already exists, in which case you can pass, but if it's some other error make sure to raise the error.